CUEstion.WEB/Controllers/QuestionsController.cs

```diff
@@ -44,31 +47,34 @@ public IActionResult GetQuestion(int questionId)
 		{
 			try
 			{
-				return Ok();
+				var question = QuestionManager.GetQuestion(questionId);
+				return Ok(question);
 			}
 			catch (Exception e)
 			{
 				return StatusCode(500, new { Message = "Server ERROR occured." });
 			}
 		}
 
-
-		[HttpDelete("{questionId}")]
-		public IActionResult DeleteQuestion(int questionId)
+		[HttpPut]
+		//[Authorize]
+		public IActionResult CreateQuestion(QuestionDTO questionDto)
 		{
 			try
 			{
-				return Ok();
+				var question = QuestionManager.CreateQuestion(questionDto, 1);
```

Comment:
userId as a parameter for QuestionManager.CreateQuestion() should be provided by controller method. So userId should be a part of questionDto object or it should be another parameter in the CreateQuestion() controller method.